sd_jwt/bin/sd_jwt

```diff
@@ -0,0 +1,276 @@
+        )
+        _sel_exp["sd_jwt_release_payload"] = sd_jwt_release_payload
+        _sel_exp["serialized_sd_jwt_release"] = serialized_sd_jwt_release
+        _sel_exp["combined_sd_jwt_sd_jwt_release"] = f"{_sel_exp['serialized_sd_jwt']}.{serialized_sd_jwt_release}"
+        _ver_data = (
+            _sel_exp["combined_sd_jwt_sd_jwt_release"],
+            ISSUER_PUBLIC_KEY,
+            ISSUER,
+            HOLDER_KEY,
+            VERIFIER,
+            _args.nonce,
+        )
+        vc = verify(*_ver_data)
+        _sel_exp["verify"] = vc
+        
+    print("# Creating the SD-JWT-Release")
+    print_repr(
+        (
+            "Payload of the SD-JWT-Release:",
+            json.dumps(_sel_exp["sd_jwt_release_payload"], indent=_args.indent)
+        )
+    )
+    print_repr(("The serialized SD-JWT-Release:", _sel_exp["serialized_sd_jwt_release"]))
+    
+    print("# Creating the Combined Presentation")
+    print_repr(("Combined Presentation:", _sel_exp["combined_sd_jwt_sd_jwt_release"]))
+    
+    print(f"Verified claims: {json.dumps(_sel_exp['verify'], indent=_args.indent)}")
+    
+    if _args.replace_examples_in:
+        from sd_jwt.replace_utils import *
+    
+        logger.info(f"Replacing the placeholders in the '{_args.replace_examples_in}' file.")
+        replacements = {
+            "example-simple_structured-sd-jwt-claims": json.dumps(
+                examples_dict["structured"]["args"][0], indent=EXAMPLE_INDENT
+            ),
+            "example-simple_structured-sd-jwt-payload": json.dumps(
+                examples_dict["structured"]["sd_jwt_payload"], indent=EXAMPLE_INDENT
+            ),
+            "example-simple_structured-sd-jwt-encoded": fill(
+                examples_dict["structured"]["combined_sd_jwt_svc"], 
+                width=EXAMPLE_MAX_WIDTH, 
+                break_on_hyphens=False
+            ),
+            "example-simple_structured-svc-payload": json.dumps(
+                examples_dict["structured"]["svc_payload"], indent=EXAMPLE_INDENT
+            ),
+            "example-simple_structured-combined-encoded": fill(
+                examples_dict["structured"]["combined_sd_jwt_sd_jwt_release"],
+                width=EXAMPLE_MAX_WIDTH,
+                break_on_hyphens=False,
+            ),
+            "example-simple_structured-release-payload": json.dumps(
+                examples_dict["structured"]["sd_jwt_release_payload"], 
+                indent=EXAMPLE_INDENT
+            ),
+            "example-simple_structured-release-encoded": fill(
+                examples_dict["structured"]["serialized_sd_jwt_release"], 
+                width=EXAMPLE_MAX_WIDTH, 
+                break_on_hyphens=False
+            ),
+            "example-simple_structured-release-combined": fill(
+                examples_dict["structured"]["combined_sd_jwt_sd_jwt_release"],
+                width=EXAMPLE_MAX_WIDTH,
+                break_on_hyphens=False,
+            ),
+        }
```

Comment:
Something's missing above - the other examples are not generated.

Reply:
Done here:
https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/a6ed737cf67d03cfd78caadddf7d77aeff880862

I'm doing a check if something is misaligned